index.html

```diff
@@ -160,6 +160,30 @@ <h2 class="about-header">About me</h2>
             </ul>
         </div> 
         <!-- work section ends       -->
+        
+        <div class="form-start">
+        <h2 class="form-header">I'm always interested in hearing about your new projects, so if you'd like to chat please get in</h2>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] kindly consider going to[ formspree](https://formspree.io/html/) and create a form that will submit data from the form and this will avoid your form to not be found 👇 
![se](https://user-images.githubusercontent.com/60146030/221374400-08004300-bce5-4cb8-bd2a-1ad527a8bc99.PNG)